core/src/main/java/hudson/triggers/SCMTrigger.java

```diff
@@ -536,7 +536,7 @@ public String getUrlName() {
         }
 
         public String getLog() throws IOException {
-            return Util.loadFile(getLogFile());
+            return Util.loadFile(getLogFile(), Charset.defaultCharset());
```

Comment:
Consistent with `new AnnotatedLargeText<>(getLogFile(), Charset.defaultCharset()` a few lines below (and `Files.readString(Util.fileToPath(logFile), Charset.defaultCharset())` some number of lines after that), and definitely a step in the right direction, but not complete. For completeness the other calls to `new StreamTaskListener(getLogFile())` in this class should also be updated to use the two-argument version that takes a `Charset` as the second argument. I wouldn't block this PR over it, but I would definitely prefer the more complete version if you have the time.

Reply:
Yes you are right, added the `Charset.defaultCharset()` also to these two missing calls.